spec.yml

```diff
@@ -1,9 +1,9 @@
 amwa_id: BCP-006
 url: https://specs.amwa.tv/bcp-006
-name: Stream Mappings
+name: Format Mappings
 status: Suite
 repo_name: bcp-006
-alt_repo_name: nmos-stream-mappings
+alt_repo_name: nmos-format-mappings
```

Comment:
What is this actually used for? I thought new references would use the `repo_name` but that this indicated a name the spec was known as before being allocated a BCP number? https://github.com/AMWA-TV/nmos-stream-mappings redirects to https://github.com/AMWA-TV/bcp-006, but https://github.com/AMWA-TV/nmos-format-mappings is a `404`.

Reply:
`alt_repo_name` is not actually used now, though could be used to automatically create redirects.  I have reverted that change.